### ISSUE TITLE
Problems booting up

### ISSUE THREAD

**Issue by davekpython** [ccrisan/motioneyeos#1676]
Hello,

My Motioneyeos boots up to the login but I cannot see it on my laptop. I have the IP address and nothing shows up.
There seems to be  a lot of failures during the boot up process. i've attached a screen shot of the last lines of code at boot up.

![20181014_172837](https://user-images.githubusercontent.com/2895571/46922517-24145d00-cfd8-11e8-8409-0d7957e49798.jpg)

I've formatted and reformatted the sd card and downloaded the latest version many times.

Hope you can help.

Thank you.

I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version
motioneyeos-raspberrypi2-20180627.img.xz

I am running motionEyeOS version: (insert your version here, e.g. 20180314).

### Board Model
I am using the following board/model: Raspberry PI 2b.

### Camera
raspberry pi noIR camera v2

I am using the following type of camera: (choose from V4L2, MMAL, Network Camera, Fast Network Camera and Simple MJPEG Camera).

My camera model is: (insert your camera model, e.g. Logitech C290).

### Network Connection
wireless

My motionEyeOS unit is connected to the network via: wifi.

### Peripherals

I am using the following peripherals that I consider relevant to this issue:

 * (e.g. USB WiFi adapter)
 * (e.g. External Hard Disk)

### Log Files

I consider the following log files relevant to this issue:

 * (attach e.g. motioneye.log)
 * (attach e.g. boot.log)

**Comment by ccrisan** [comment]
Please try to take a picture of the beginning of the log, **at the very first boot** after writing the OS image. Your data partition is not being created and that results in all the painful errors you see.

You could "scroll" through the console log from an attached keyboard using Shift + Page Up/Down.

**Comment by davekpython** [comment]
Yes, I think that's correct. It seems it's not creating a lot of directories. (here's a photo at top).

Thank you.

![20181015_101854](https://user-images.githubusercontent.com/2895571/46957234-22dd4180-d065-11e8-968f-e2adca38d45b.jpg)

**Comment by ccrisan** [comment]
This seems like a photo from a subsequent failing boot. I need one from the very first boot right after you have written the OS image from scratch to your card.

**Comment by davekpython** [comment]
In an effort to problem solve, I tried reformatting the SD Card and found it to be corrupted.

So I just picked up a new Sandisk Extreme 32gb sd card. 

I did not format it as it was already formatted. The win32disk imager copied the motioneyeos-raspberrypi2-20180627 img to the sd card. I extracted fwupdater zip to get start.elf and rootfs.cpio.  Then put the sd card into the raspberry pi b2. I attached the raspberry pi to the router with an ethernet cord and booted up the sd card.  (see below image for screen shot.

It seems like it should be working. But, I can't "see" my rasp pi ip address. I know what it is but it doesn't show up on my angry ip scanner (it used to) and when I put the address in the browser it doesn't find it.

I did notice there is a kernal img folder as one of the files on the sd card (see image below) I wonder if this might have something to do with the problem. should i have copied the img file to my computer then used the windisc imager to extract it to the card?

Thoughts?

Thank you.

![extracted fwupdater zip to get start elf and rootfs cpio](https://user-images.githubusercontent.com/2895571/46979077-994a6580-d09e-11e8-9c77-b84d374ea981.png)


![20181015_171101](https://user-images.githubusercontent.com/2895571/46978763-cba79300-d09d-11e8-923b-6cfed17702d6.jpg)

**Comment by davekpython** [comment]
Update.

I'm happy to say I have it up and running (the new sd card made the difference... mostly.

I have pointed my browser to the pi's ip address and my motioneye page is right there but there is no image from the camera. see photo 
![2018-10-15 19_32_13-meye-fdaad03a up and running no camera feed](https://user-images.githubusercontent.com/2895571/46984119-b3417380-d0b1-11e8-8898-f0aa39cd7da3.png)


I installed the camera before working with motioneye so i know it works.

thoughts?

Thank you.
glad to be getting close.
